changelogs/fragments/ansible-base-update-containers.yml

```diff
@@ -1,5 +1,5 @@
 minor_changes:
   - ansible-test - OpenSuse container now uses Leap 15.2 (https://github.com/ansible/distro-test-containers/pull/48).
-  - ansible-test - Ubuntu containers as well as ``default-test-container`` and ``ansible-base-test-container`` are now slightly smaller due to apt cleanup (https://github.com/ansible/distro-test-containers/pull/46).
+  - ansible-test - Ubuntu containers as well as ``default-test-container`` and ``ansible-core-test-container`` are now slightly smaller due to apt cleanup (https://github.com/ansible/distro-test-containers/pull/46).
   - ansible-test - CentOS 8 container is now 8.2.2004 (https://github.com/ansible/distro-test-containers/pull/45).
-  - ansible-test - ``default-test-container`` and ``ansible-base-test-container`` now use Python 3.9.0 instead of 3.9.0rc1.
+  - ansible-test - ``default-test-container`` and ``ansible-core-test-container`` now use Python 3.9.0 instead of 3.9.0rc1.
```

Comment:
This one too.  When we rename the container, I imagine we'll create a new fragment that indicates the container was renamed.

Reply:
Fixed